src/ui_interface.rs

```diff
@@ -243,7 +243,11 @@ pub fn set_peer_option(id: String, name: String, value: String) {
 
 #[inline]
 pub fn using_public_server() -> bool {
-    crate::get_custom_rendezvous_server(get_option_("custom-rendezvous-server")).is_empty()
+    if hbb_common::config::RS_PUB_KEY == "OeVuKk5nlHiXp+APNn0Y3pC1Iwpwn44JGqrQCsWqmBw=" {
+        return true
+    } else {
+        return false
+    }
```

Comment:
I do not think. this is necessary.

Reply:
So I just tested and verified, with the code the way it was before it put that message at the bottom when using github actions to build at least.